napalm/eos/eos.py

```diff
@@ -1682,13 +1682,26 @@ def _parse_per_peer_bgp_detail(peer_output):
                 "configured_keepalive",
                 "advertised_prefix_count",
                 "received_prefix_count",
+                "advertised_ipv6_prefix_count",
+                "received_ipv6_prefix_count",
             ]
 
             peer_details = []
 
+            # determine if in multi-agent mode to get correct extractor_type
+            commands = [
+                "show running-config | include service routing protocols model multi-agent"
+            ]
+            is_multi_agent = self.device.run_commands(commands, encoding="text")[0].get(
```

Comment:
Can probably get rid of the `commands` variable:

```suggestion
            is_multi_agent = self.device.run_commands(
                [
                    "show running-config | include service routing protocols model multi-agent"
                ],
                encoding="text",
            )[0].get()
```